kinto/tests/test_views_collections.py

```diff
@@ -35,12 +35,15 @@ def test_collections_name_should_be_simple(self):
                           headers=self.headers,
                           status=400)
 
-    def test_unknown_bucket_raises_404(self):
+    def test_unknown_bucket_raises_403(self):
```

Comment:
Here, we are returning a 403 if the resource doesn't exist. When a resource doesn't exist, a 404 should be returned. 403 means that the resource exists but that you cannot access it.